tests/Core/List/FluentPersonaTests.cs

```diff
@@ -29,14 +29,47 @@ public void FluentPersona_Image()
         cut.Verify();
     }
 
+    [Theory]
+    [InlineData("Denis Voituron")]
+    [InlineData("Denis   Voituron")]
+    [InlineData("Denis   Voituron  Senior")]
+    public void FluentPersona_Initials(string name)
+    {
+        // Arrange
+        var cut = TestContext.RenderComponent<FluentPersona>(parameters =>
+        {
+            parameters.Add(p => p.Id, "myComponent");
+            parameters.Add(p => p.Name, name);
+            parameters.Add(p => p.DismissTitle, "Remove this people");
+        });
+
+        // Assert
+        cut.Verify(suffix: name);
```

Comment:
To avoid very long filename like `FuentPersona_Initials-Denis[space][space][space]Voituron[space][space]Senior.verified.html`

Could you add another argument like 'id' with values like:
- OneSpace
- MultipleSpace
- MultipleWords
- OneName ²
- NoName ²

² You could also merge the `FluentPersona_InitialsNoName` and `FluentPersona_InitialsNoName` methods into this new method.